consensus/ibft/src/test/java/tech/pegasys/pantheon/consensus/ibft/validation/RoundChangeMessageValidatorTest.java

```diff
@@ -201,15 +200,14 @@ public void roudnChangeWithPastPreprepareForCurrentHeightIsSuccessful() {
         proposerMessageFactory.createIbftSignedRoundChangeMessageData(
             targetRound, Optional.of(prepareCertificate));
 
-    when(basicValidator.addPreprepareMessage(prepareCertificate.getIbftPrePrepareMessage()))
+    when(basicValidator.addPreprepareMessage(prepareCertificate.getProposalMessage()))
         .thenReturn(true);
     when(basicValidator.validatePrepareMessage(prepareMsg)).thenReturn(true);
 
     assertThat(validator.validateMessage(msg)).isTrue();
     verify(validatorFactory, times(1))
-        .createAt(prepareCertificate.getIbftPrePrepareMessage().getPayload().getRoundIdentifier());
-    verify(basicValidator, times(1))
-        .addPreprepareMessage(prepareCertificate.getIbftPrePrepareMessage());
+        .createAt(prepareCertificate.getProposalMessage().getPayload().getRoundIdentifier());
+    verify(basicValidator, times(1)).addPreprepareMessage(prepareCertificate.getProposalMessage());
```

Comment:
is addPreprepareMessage something that should be changed?

Reply:
refactor is not your friend - it should just be addPrepareMessage.